main.py

```diff
@@ -37,6 +40,17 @@ def __init__(self):
 
     def _run(self):
         for submission in self.reddit.subreddit(self.config.SUBREDDIT).stream.submissions():
+            
+            #start by checking if any posts posted 70+ hours ago have broken 100 upvotes, if not hit 100 delete to not check it again
```

Comment:
Probably better to do this with a separate timer function rather than on each new post.  Perhaps have it run every 10-15 minutes?

Reply:
Agree, I'm just too incomptent in python

Reply:
I've done some research on this and python, looks like in order to do this will have to go into multithreading, which again causes a whole mess with sqlite conflicts. Considering how inexpensive the function for checking for successful posts is, I think it's way more work than value. Going to resolve.